man/sigrok_rdm_sniffer.1

```diff
@@ -0,0 +1,48 @@
+.TH sigrok_rdm_sniffer 1 "March 2021"
+.SH NAME
+sigrok_rdm_sniffer \- Decode DMX/RDM data from devices supported by sigrok
+.SH SYNOPSIS
+.B sigrok_rdm_sniffer
+[ options ]
+.SH DESCRIPTION
+.B sigrok_rdm_sniffer
+Decode DMX/RDM data from devices supported by sigrok
+.SH OPTIONS
+.IP "-d, --display-dmx"
+Display DMX Frames. Defaults to false.
+.IP "-h, --help"
+Display the help message
+.IP "-l, --log-level <int8_t>"
+Set the logging level 0 .. 4.
+.IP "-r, --full-rdm"
+Unpack RDM parameter data.
+.IP "-t, --timestamp"
```

Comment:
Is that implemented? Doesn't seem to work with just `-d -t`. Maybe it's just for RDM frames?

Reply:
No, I found out some time back it wasn't implemented on the original Logic but left is present just in case it broke people's CLIs:
https://github.com/peternewman/ola/blob/sigrok/tools/logic/logic-rdm-sniffer.cpp#L86-L87

Obviously that appeared when I duplicated this code. I've commented it out for now entirely in the sigrok based sniffer. When it all gets pulled out to some common code I'll try and finally implement it too, then they'll both get it!